go-controller/pkg/ovn/ovn.go

```diff
@@ -318,6 +322,10 @@ func (oc *Controller) Run(wg *sync.WaitGroup) error {
 
 	klog.Infof("Completing all the Watchers took %v", time.Since(start))
 
+	// Remove old multicastDefaultDeny port group now that all ports
+	// have been added to the clusterPortGroup by WatchPods()
+	deletePortGroup("mcastPortGroupDeny")
```

Comment:
Move this into createDefaultDenyMulticastPolicy()

Reply:
Fixed.